.github/workflows/build.yml

```diff
@@ -35,6 +35,11 @@ jobs:
   test:
     runs-on: ubuntu-latest
     timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        # used by the jupyterlab/maintainer-tools base-setup action
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
Should we test on all Python versions? Or maybe the lowest and highest versions (and 3.12) would be enough, like here?

https://github.com/jupyter/notebook/blob/76db867671eb43de5f5fc6adbb8be8c52b9ae36e/.github/workflows/build.yml#L121

Reply:
This is mostly to help reduce the number of CI jobs if they are not strictly necessary.

Reply:
It's up to you, but #7014 would not have been caught by your proposed set of pythons:

There was 

```python
importlib-resources>=5.0;python_version<\"3.9\"",
```

and the failure was only with Python 3.9. For 3.8 we already have the backport and for python 3.10 the included stdlib is okay.

Reply:
Ah ok then we can keep all versions and see how it goes with CI over time.